### ISSUE TITLE
Sponsor not displayed as active despite a sponsor.yaml being present

### ISSUE THREAD

**Issue by D4N** [FrostyX/fedora-sponsors#13]
The current sponsor page does not display me as an active sponsor (fas: defolos), despite me having a sponsor.yaml on fedorapeople.org (https://defolos.fedorapeople.org/sponsor.yaml). what's even weirder is that the backend appears to have processed me correctly, as my languages and areas of interest are correct. I'm just marked as inactive for some reason.

**Comment by FrostyX** [comment]
Hello @D4N,
thank you for the feedback.

To decide whether a sponsor is showed as active or inactive, we use this [activity.py](https://github.com/FrostyX/fedora-sponsors/blob/main/activity.py) script. It is an old proof-of-concept script from @xsuchy and it would deserve improvements for sure. The idea is - if there is any activity on any package review BZ in the last two years, the sponsor is considered to be active. Is this the case for you? There might be some bug in the script ...

The output from today

```
python activity.py
Aurelien Bompard <abompard> - no recent sponsor activity
Adrien Devresse <adev> - no recent sponsor activity
Adrian Reber <adrian> - no recent sponsor activity
Alexander Kurtakov <akurtakov> - no recent sponsor activity
Alexander Larsson <alexl> - no recent sponsor activity
Ankur Sinha (FranciscoD) <ankursinha> worked on BZ 1809405
Ankur Sinha (FranciscoD) <ankursinha> worked on BZ 1816301
Ankur Sinha (FranciscoD) <ankursinha> worked on BZ 1832562
Ankur Sinha (FranciscoD) <ankursinha> worked on BZ 1849706
Ankur Sinha (FranciscoD) <ankursinha> worked on BZ 1923830
Ankur Sinha (FranciscoD) <ankursinha> - directly sponsored: ['mhough', 'alciregi', 'bt0dotninja', 'major', 'jlanda', 'tronde', 'mikelo2', 'pampelmuse', 'iztokf', 'nerdsville']
Athos Ribeiro <athoscr> worked on BZ 1442576
Athos Ribeiro <athoscr> worked on BZ 1463492
Dennis Gilmore <ausil> - no recent sponsor activity
Andrea Veri <averi> - no recent sponsor activity
Andreas Bierfert <awjb> - no recent sponsor activity
Daniel Berrangé <berrange> - directly sponsored: ['etrunko']
Björn 'besser82' Esser <besser82> - no recent sponsor activity
Slavek Kabrda <bkabrda> - no recent sponsor activity
Randy Barlow <bowlofeggs> - no recent sponsor activity
Christian Iseli <c4chris> - no recent sponsor activity
Christopher Aillon (sabbatical, not receiving bugmail) <caillon> - no recent sponsor activity
Carl George 🤠 <carlwgeorge> - directly sponsored: ['gcabiddu', 'yogaraj']
Chitlesh GOORAH <chitlesh> - no recent sponsor activity
Miro Hrončok <churchyard> - directly sponsored: ['rhcontainerbot', 'marxin']
Rick Elrod <codeblock> - directly sponsored: ['phuzion']
Ralf Corsepius <corsepiu> - no recent sponsor activity
Cole Robinson <crobinso> - directly sponsored: ['alwillia', 'mrezanin']
Chris Weyl <cweyl> - no recent sponsor activity
Christoph Wickert <cwickert> - no recent sponsor activity
David Cantrell <dcantrell> worked on BZ 1822971
David Cantrell <dcantrell> - directly sponsored: ['nickblack']
Fabio Valentini <decathorpe> - directly sponsored: ['astra']
 <defolos> - no recent sponsor activity
Denis Leroy <denis> - no recent sponsor activity
Dave Malcolm <dmalcolm> - no recent sponsor activity
Douglas Schilling Landgraf <dougsland> - no recent sponsor activity
David Woodhouse <dwmw2> - no recent sponsor activity
Eduardo Echeverria <echevemaster> - no recent sponsor activity
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1519834
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1535549
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1554020
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1554021
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1554022
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1652588
Robert-André Mauchin 🐧 <eclipseo> worked on BZ 1654876
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1658153
Robert-André Mauchin 🐧 <eclipseo> worked on BZ 1662170
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1663633
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1692560
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1718339
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1719956
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1719957
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1719959
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1734161
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1735762
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1746017
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1747552
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1752139
Robert-André Mauchin 🐧 <eclipseo> worked on BZ 1752158
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1752233
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1756901
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1756905
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1756906
Robert-André Mauchin 🐧 <eclipseo> worked on BZ 1756907
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1756908
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1758036
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1758487
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1767998
Robert-André Mauchin 🐧 <eclipseo> worked on BZ 1773300
Robert-André Mauchin 🐧 <eclipseo> worked on BZ 1773855
Robert-André Mauchin 🐧 <eclipseo> worked on BZ 1778512
Robert-André Mauchin 🐧 <eclipseo> worked on BZ 1780297
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1786724
Robert-André Mauchin 🐧 <eclipseo> worked on BZ 1795081
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1808278
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1811173
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1811174
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1811177
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1811179
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1811180
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1811182
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1811183
Robert-André Mauchin 🐧 <eclipseo> worked on BZ 1811265
Robert-André Mauchin 🐧 <eclipseo> worked on BZ 1811266
Robert-André Mauchin 🐧 <eclipseo> worked on BZ 1812411
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1812961
Robert-André Mauchin 🐧 <eclipseo> worked on BZ 1813737
Robert-André Mauchin 🐧 <eclipseo> worked on BZ 1818670
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1837107
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1842225
Robert-André Mauchin 🐧 <eclipseo> removed FE-NEEDSPONSOR from BZ 1843300
Robert-André Mauchin 🐧 <eclipseo> worked on BZ 1846749
Robert-André Mauchin 🐧 <eclipseo> - directly sponsored: ['kochstefan', 'pschindl', 'korkeala', 'paulcarroty', 'jcapitao', 'jbertozzi', 'mufti11', 'muneendra', 'wef', 'jrincayc', 'davidsch', 'jelford', 'lyessaadi', 'wberrier', 'djw8605', 'mhjacks', 'lnie', 'yonhan', 'jbaublitz', 'rffontenelle', 'andymenderunix', 'adsr', 'kmabbasi', 'v02460', 'souryogurt', 'ibotty', 'luigifab', 'jijunwan']
Enrico Scholz <ensc> - no recent sponsor activity
Fabio Alessandro Locati <fale> - no recent sponsor activity
Thomas Fitzsimmons <fitzsim> - no recent sponsor activity
Gérard Milmeister <gemi> - no recent sponsor activity
Marek Goldmann <goldmann> - no recent sponsor activity
Haïkel Guémar <hguemar> - directly sponsored: ['mhuin']
Richard Shaw <hobbes1069> - directly sponsored: ['nickysn', 'lu3vea']
Iain Arnell <iarnell> - no recent sponsor activity
Igor Raits <ignatenkobrain> removed FE-NEEDSPONSOR from BZ 1905143
Igor Raits <ignatenkobrain> removed FE-NEEDSPONSOR from BZ 1905145
Igor Raits <ignatenkobrain> removed FE-NEEDSPONSOR from BZ 1905148
Igor Raits <ignatenkobrain> - directly sponsored: ['tjmc', 'julian8628', 'directhex', 'lgoncalv', 'darkmuggle', 'ckuehl']
Ignacio Vazquez-Abrams <ivazquez> - no recent sponsor activity
Andreas Thienemann <ixs> - no recent sponsor activity
José Matos <jamatos> - no recent sponsor activity
Severin Gehwolf <jerboaa> - directly sponsored: ['jhuttana']
James Hogarth <jhogarth> - no recent sponsor activity
Jerry James <jjames> - directly sponsored: ['yanqiyu']
John (J5) Palmieri <johnp> - no recent sponsor activity
Jose Pedro Oliveira <jpo> - no recent sponsor activity
Jaroslav Škarvada <jskarvad> - directly sponsored: ['aegorenk', 'hony', 'japokorn', 'reznik', 'jkacur', 'jzerdik', 'ggasparb']
Jef Spaleta <jspaleta> - no recent sponsor activity
Susi Lehtola <jussilehtola> - no recent sponsor activity
jiri vanek <jvanek> worked on BZ 1804257
jiri vanek <jvanek> - directly sponsored: ['jandrlik']
Jarod Wilson <jwilson> - no recent sponsor activity
Hans de Goede <jwrdegoede> worked on BZ 1884608
Hans de Goede <jwrdegoede> - directly sponsored: ['pbo']
Kalev Lember <kalev> - no recent sponsor activity
Karsten <karsten> - no recent sponsor activity
Jeremy Katz <katzj> - no recent sponsor activity
David Nalley <ke4qqq> - no recent sponsor activity
Kevin Kofler <kkofler> - directly sponsored: ['andreamtp']
Ken Dreyer <ktdreyer> - directly sponsored: ['adeza', 'jcaratzas', 'prongs', 'timothyasir']
Xavier Lamien <laxathom> - no recent sponsor activity
Lumír Balhar <lbalhar> - directly sponsored: ['ksurma']
Luis Bazan <lbazan> - no recent sponsor activity
Lukas Berk <lberk> - directly sponsored: ['amerey', 'sapatel', 'almac']
leigh scott <leigh123linux> - no recent sponsor activity
Gwyn Ciesla <limb> - no recent sponsor activity
Lubomir Rintel <lkundrak> - directly sponsored: ['benesv']
lutter@watzmann.net <lutter> - no recent sponsor activity
Mark McLoughlin <markmc> - no recent sponsor activity
Matthew Miller <mattdm> - no recent sponsor activity
Adam Miller <maxamillion> - no recent sponsor activity
Eduardo Mayorga <mayorga> - no recent sponsor activity
Mat Booth <mbooth> - no recent sponsor activity
Matthias Clasen <mclasen> - no recent sponsor activity
Matthew Booth <mdbooth> - no recent sponsor activity
Martin Gieseking <mgieseki> - no recent sponsor activity
Michal Schmidt <michich> - directly sponsored: ['lsun', 'iivanov', 'izabela']
Michael S. <misc> - directly sponsored: ['rkothiya']
Mikolaj Izdebski <mizdebsk> - no recent sponsor activity
Matthias Runge <mrunge> - directly sponsored: ['chrissie']
Michael Schwendt <mschwendt> - no recent sponsor activity
Miroslav Suchý <msuchy> worked on BZ 1748233
Miroslav Suchý <msuchy> removed FE-NEEDSPONSOR from BZ 1758499
Miroslav Suchý <msuchy> worked on BZ 1981739
Miroslav Suchý <msuchy> worked on BZ 1748233
Miroslav Suchý <msuchy> removed FE-NEEDSPONSOR from BZ 1758499
Miroslav Suchý <msuchy> worked on BZ 1981739
Miroslav Suchý <msuchy> - directly sponsored: ['ljavorsk', 'fjanus', 'amatej', 'mfabik', 'mgrabovs', 'thrnciar', 'ogajduse', 'nsella', 'csomh', 'sakalosj', 'lbarczio', 'mcurlej']
Mamoru TASAKA <mtasaka> - no recent sponsor activity
Nathan Scott <nathans> worked on BZ 1919347
Nathan Scott <nathans> worked on BZ 1919349
Nathan Scott <nathans> - directly sponsored: ['agerstmayr', 'portante', 'dhat180']
Nick Bebout <nb> - directly sponsored: ['crummel', 'cyberboxster', 'gonis', 'isaact', 'calexandru2018']
Neal Gompa <ngompa> worked on BZ 1401365
Neal Gompa <ngompa> removed FE-NEEDSPONSOR from BZ 1766375
Neal Gompa <ngompa> removed FE-NEEDSPONSOR from BZ 1770566
Neal Gompa <ngompa> removed FE-NEEDSPONSOR from BZ 1780882
Neal Gompa <ngompa> removed FE-NEEDSPONSOR from BZ 1787714
Neal Gompa <ngompa> removed FE-NEEDSPONSOR from BZ 1816124
Neal Gompa <ngompa> removed FE-NEEDSPONSOR from BZ 1886621
Neal Gompa <ngompa> removed FE-NEEDSPONSOR from BZ 1920606
Neal Gompa <ngompa> removed FE-NEEDSPONSOR from BZ 1993632
Neal Gompa <ngompa> removed FE-NEEDSPONSOR from BZ 1993634
Neal Gompa <ngompa> - directly sponsored: ['dmccheyne', 'brejoc', 'asaleh', 'rhea', 'raineforest', 'lkiesow', 'acardace', 'appadeia', 'haircommander', 'oden', 'fellipeh', 'egustavs', 'sturivny', 'esyr', 'daxelrod', 'acui', 'patrickt', 'siosm', 'ericedens', 'kir']
Neil Horman <nhorman> worked on BZ 1785976
Neil Horman <nhorman> - directly sponsored: ['trix', 'aravuri', 'nkorde', 'vladis', 'avijayan']
Nicolas Mailhot <nim> - no recent sponsor activity
Mukundan Ragavan <nonamedotc> worked on BZ 1636111
Mukundan Ragavan <nonamedotc> worked on BZ 1636111
Mukundan Ragavan <nonamedotc> - directly sponsored: ['blowry', 'taaem', 'music']
Bill Nottingham <notting> - no recent sponsor activity
Nils Philippsen <nphilipp> worked on BZ 1947670
Orcan Ogetbil <oget> - no recent sponsor activity
Orion Poplawski <orion> removed FE-NEEDSPONSOR from BZ 1856005
Orion Poplawski <orion> - directly sponsored: ['pgrosu', 'jngrad']
T.C. Hollingsworth <patches> - no recent sponsor activity
Peter Robinson <pbrobinson> - directly sponsored: ['njha', 'torbuntu', 'mzavalavz', 'chimosky', 'jlinton', 'gtiwari']
Paulo Andrade <pcpa> - no recent sponsor activity
Patrice Dumas <pertusus> - no recent sponsor activity
Peter Lemenkov <peter> - directly sponsored: ['luismartingil']
Jens Petersen <petersen> - no recent sponsor activity
Paul Howarth <pghmcfc> - directly sponsored: ['dreua']
Pierre-YvesChibon <pingou> - directly sponsored: ['packagerbot', 'scoady']
Pris Nasrat <pnasrat> - no recent sponsor activity
Parag Nemade <pnemade> - no recent sponsor activity
Petr Šabata <psabata> - no recent sponsor activity
Patrick Uiterwijk <puiterwijk> - no recent sponsor activity
Paul Wouters <pwouters> - no recent sponsor activity
Ralph Bean <ralph> - directly sponsored: ['breilly', 'martstyk']
Dominik 'Rathann' Mierzejewski <rathann> - directly sponsored: ['xdelaruelle']
Rex Dieter <rdieter> - directly sponsored: ['thunderbirdtr']
Remi Collet <remi> - no recent sponsor activity
Debarshi Ray <rishi> - no recent sponsor activity
Richard W.M. Jones <rjones> - no recent sponsor activity
Robert Scheck <robert> - no recent sponsor activity
Ryan O'Hara <rohara> worked on BZ 1773716
Ryan O'Hara <rohara> worked on BZ 1773717
Ryan O'Hara <rohara> worked on BZ 1773718
Ryan O'Hara <rohara> worked on BZ 1773719
Ryan O'Hara <rohara> worked on BZ 1773720
Ryan O'Hara <rohara> worked on BZ 1833511
Ryan O'Hara <rohara> - directly sponsored: ['bdperkin']
Ruben Kerkhof <ruben> - no recent sponsor activity
Radek Vokál <rvokal> - no recent sponsor activity
Jochen Schmitt <s4504kr> - no recent sponsor activity
Michel Alexandre Salim <salimma> worked on BZ 1804307
Michel Alexandre Salim <salimma> worked on BZ 1839330
Michel Alexandre Salim <salimma> worked on BZ 1891963
Michel Alexandre Salim <salimma> worked on BZ 1933459
Michel Alexandre Salim <salimma> worked on BZ 1933462
Michel Alexandre Salim <salimma> worked on BZ 1953243
Michel Alexandre Salim <salimma> - directly sponsored: ['eeickmeyer', 'returntrip', 'dcavalca', 'malmond', 'jaymzh']
Eric Sandeen <sandeen> - directly sponsored: ['ronniesahlberg']
Ville Skyttä <scop> - no recent sponsor activity
Steven Dake <sdake> - no recent sponsor activity
Stephen Gallagher <sgallagh> - directly sponsored: ['thalman', 'iboukris', 'atikhonov', 'kevinb', 'kfan']
Dan Horák <sharkcz> - directly sponsored: ['tuliom', 'stliibm', 'alexsaezm', 'sfroemer', 'nnac123']
Simone Caronni <slaanesh> worked on BZ 1749383
Simone Caronni <slaanesh> - directly sponsored: ['jromanes']
Stanislav Ochotnicky <sochotni> - no recent sponsor activity
Steven Pritchard <steve> - no recent sponsor activity
Steve Traylen <stevetraylen> - no recent sponsor activity
Ben Rosser <tc01> - no recent sponsor activity
Matthias Saou <thias> - no recent sponsor activity
Thorsten Leemhuis <thl> - no recent sponsor activity
Thomas Vander Stichele <thomasvs> - no recent sponsor activity
Till Maas <till> - directly sponsored: ['ffmancera', 'norbertus']
Tim Lauridsen <timlau> - no recent sponsor activity
Tomáš Mráz <tmraz> - directly sponsored: ['saprasad', 'zpytela', 'alakatos', 'gtb', 'ipedrosa', 'evgenyz', 'vpolasek', 'zfridric', 'mlysonek', 'asosedkin', 'szidek', 'dbelyavs']
Thomas Spura <tomspur> - no recent sponsor activity
Tomas Orsava <torsava> - directly sponsored: ['akhaitov']
Toshio Ernie Kuratomi <toshio> - no recent sponsor activity
Steven M. Parrish <tuxbrewr> - no recent sponsor activity
Tim Waugh <twaugh> - no recent sponsor activity
Vasiliy Glazov <vascom> worked on BZ 1833612
Vasiliy Glazov <vascom> worked on BZ 1833785
Vasiliy Glazov <vascom> worked on BZ 1840134
Vasiliy Glazov <vascom> worked on BZ 1840865
Vasiliy Glazov <vascom> worked on BZ 1844120
Vasiliy Glazov <vascom> removed FE-NEEDSPONSOR from BZ 1917128
Vasiliy Glazov <vascom> worked on BZ 1936414
Vasiliy Glazov <vascom> - directly sponsored: ['elxreno', 'girst', 'meeuw', 'zawertun', 'yshestakov', 'vladius', 'alexfails']
Volker Fröhlich <volter> - no recent sponsor activity
Vít Ondruch <vondruch> - directly sponsored: ['vmihalko', 'opohorel']
Wart <wart> - no recent sponsor activity
Will Benton <willb> - no recent sponsor activity
William Moreno <williamjmorenor> - no recent sponsor activity
manuel wolfshant <wolfy> - no recent sponsor activity
Warren Togami <wtogami> - no recent sponsor activity
Xavier Bachelot <xavierb> - directly sponsored: ['ikedas']
Zbigniew Jędrzejewski-Szmek <zbyszek> worked on BZ 1358959
Zbigniew Jędrzejewski-Szmek <zbyszek> worked on BZ 1885684
Zbigniew Jędrzejewski-Szmek <zbyszek> - directly sponsored: ['lcts', 'jimbair', 'obudai', 'mrc0mmand', 'dvdhrm', 'abiagion', 'lijiang', 'sohank2602', 'fepitre', 'oturpe', 'yuwata']
```

**Comment by D4N** [comment]
Jakub Kadlčík ***@***.***> writes:

>  <defolos> - no recent sponsor activity

This is weird, because I sponsored ivzhh just a few days ago into the
packager group.

**Comment by FrostyX** [comment]
Okay, thank you @D4N,
I will investigate. There will probably be something wrong with the script.

**Comment by FrostyX** [comment]
@D4N can you please link the ticket you sponsored, so I can easily test my WIP changes?

**Comment by D4N** [comment]
I have simply added the contributor to the packager group, I did not do
it via a ticket. I guess that's not how it should be done then?

**Comment by FrostyX** [comment]
> I have simply added the contributor to the packager group, I did not do it via a ticket.

AFAIK this should be recognized as well. I made the following reproducer based on the `activity.py` code. You can try it yourself.

```python
import getpass
from fedora.client import AccountSystem

client = AccountSystem()
client.username = input('Username: ').strip()
client.password = getpass.getpass('Password: ')

user = client.person_by_username("defolos")
group = client.group_by_name("packager")
for role in group.approved_roles:
    if role.sponsor_id == user.id:
        print("{0} was sponsored by {1}".format(
            client.person_by_id(role.person_id).username,
            user.username))
```

Unfortunately, it doesn't return any results. When I change "defolos" to e.g. "msuchy", I get a lot of results including myself among the sponsorees. Do you know what may be wrong here?

**Comment by FrostyX** [comment]
I fixed the issue in PR #14.

There was a condition like so (meaning, the sponsor was skipped and automatically considered inactive)

```python
if not user.human_name:
    return None
```

Its value was taken from bugzilla

```python
return self.bz.getuser(self.fas.bugzilla_email).real_name
```

I am not sure if you don't have your real name configured in Bugzilla or there was some other error but it sounded reasonable to me to just obtain the real name from FAS instead. Your name is there configured correctly, so I rebuild the page and you are now displayed as active.

I also made a change, that everyone who has `sponsor.yaml` file on fedorapeople.org is considered active.

**Comment by D4N** [comment]
Jakub Kadlčík ***@***.***> writes:

> I fixed the issue in PR #14.

Thank you!

> There was a condition like so (meaning, the sponsor was skipped and automatically considered inactive)
>
> ```python
> if not user.human_name:
>     return None
> ```

Huh, turns out I completely forgot to set this field.
